### PR TITLE
Changes to adapt to R2 cluster configurations

### PR DESCRIPTION
**Pull request by josephthaliath**

### CODE REVIEW

File: workloads/oai/README.md [hunk 1]
```diff
@@ -1,9 +1,98 @@
...
+
+# Steps to deploy OAI RAN on Nephio
+
+## Step 0: Prerequisite
+
+1. Nephio should be installed.
+2. Verify Network required for OAI RAN and Core should should be installed.
+   Perform the following to verify in the Nephio management cluster.
+3. Catalog repository need to be registered. 
+   Since the OAI RAN packages are in workloads/oai of https://github.com/josephthaliath/catalog.git repository,
```
Comment (johnbelamaric) [inline workloads/oai/README.md:23]: These should reference the nephio-project org, not your personal github.

File: workloads/oai/pkg-example-cucp-bp/apply-replacements-owner.yaml [hunk 2]
```diff
@@ -24,20 +24,6 @@ replacements:
     options:
       delimiter: '-'
```
Comment (johnbelamaric) [inline workloads/oai/pkg-example-cucp-bp/apply-replacements-owner.yaml:25]: I have found apply-replacements with delimiters to be very fragile, I would avoid them if possible (maybe a future update, or eventually I would hope these become unnecessary with the implementation of CEL for function inputs)

**Review by johnbelamaric — COMMENTED** [review]
I think just the README needs to be updated to reference the nephio-project org.

After this merges, I can create the tags.

File: workloads/oai/pkg-example-cucp-bp/apply-replacements-owner.yaml (hunk 2 above)
Comment (josephthaliath) [reply workloads/oai/pkg-example-cucp-bp/apply-replacements-owner.yaml:25]: Sure @johnbelamaric , we will consider it in the next updates.

**Comment by johnbelamaric** [comment]
Tags are done:

```
[hi on] jbelamaric@jbelamaric:~/proj/gh/nephio-project/catalog$ git tag workloads/oai/oai-ran-operator/v1
[hi on] jbelamaric@jbelamaric:~/proj/gh/nephio-project/catalog$ git tag workloads/oai/pkg-example-cucp-bp/v1
[hi on] jbelamaric@jbelamaric:~/proj/gh/nephio-project/catalog$ git tag workloads/oai/pkg-example-cuup-bp/v1
[hi on] jbelamaric@jbelamaric:~/proj/gh/nephio-project/catalog$ git tag workloads/oai/pkg-example-du-bp/v1
[hi on] jbelamaric@jbelamaric:~/proj/gh/nephio-project/catalog$ git tag workloads/oai/pkg-example-ue-bp/v1
[hi on] jbelamaric@jbelamaric:~/proj/gh/nephio-project/catalog$ git tag
distros/gcp/cc-rootsync/v1
distros/gcp/nephio-mgmt/v1
infra/gcp/cc-cluster-gke-std-csr-cs/v1
infra/gcp/cc-repo-csr/v1
infra/gcp/nephio-blueprint-repo/v1
infra/gcp/nephio-workload-cluster-gke/v1
workloads/oai/oai-ran-operator/v1
workloads/oai/pkg-example-cucp-bp/v1
workloads/oai/pkg-example-cuup-bp/v1
workloads/oai/pkg-example-du-bp/v1
workloads/oai/pkg-example-ue-bp/v1
[hi on] jbelamaric@jbelamaric:~/proj/gh/nephio-project/catalog$ git push --tags
Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
To github.com:nephio-project/catalog.git
 * [new tag]         workloads/oai/oai-ran-operator/v1 -> workloads/oai/oai-ran-operator/v1
 * [new tag]         workloads/oai/pkg-example-cucp-bp/v1 -> workloads/oai/pkg-example-cucp-bp/v1
 * [new tag]         workloads/oai/pkg-example-cuup-bp/v1 -> workloads/oai/pkg-example-cuup-bp/v1
 * [new tag]         workloads/oai/pkg-example-du-bp/v1 -> workloads/oai/pkg-example-du-bp/v1
 * [new tag]         workloads/oai/pkg-example-ue-bp/v1 -> workloads/oai/pkg-example-ue-bp/v1
[hi on] jbelamaric@jbelamaric:~/proj/gh/nephio-project/catalog$ 

```

**Pull request merged**